### ISSUE TITLE
Users: show also projects of user

### ISSUE THREAD

**Issue by HeinzWuert** [kevinpapst/kimai2#2719]
When double clicking a user his statistics and belonging to teams are shown.
Looks good!
Perfect would be to display all projects where this specific user is working in.

This function makes only sense for Admins!

**Comment by kevinpapst** [comment]
Can you dd some more details?
How do we identify these projects?
By checking the past records? 
Are you interested in statistics or do you want to see the permissions (list of projects of all teams)?

**Comment by HeinzWuert** [comment]
Yes, but it's for use of admins to get an quick overview in which teams/projects the user is involved.

Additionally a pulldown button in every row to enable the deleting of team membership

**Comment by blaue-mango** [comment]
I second this.

Regarding implementation I would prefer a view in which all the active projects are listed in which the user has at least one activity in it. 

This could also just extend the already existing projects view by adding a filter for "user" which will filter the projects view for just the projects the user has worked in.
